nginx_proxy/config.yaml

```
@@ -37,5 +40,8 @@ schema:
   cloudflare: bool
   customize:
     active: bool
-    default: str
-    servers: str
+    default: str?
+    servers: str?
```

These should remain mandatory. If a default value is set, they automatically become mandatory settings. The user doesn't need to provide them, since there is a default, but from a config perspective, they are mandatory.

```suggestion
    default: str
    servers: str
```

The docs are a bit unclear, but I tested the behavior (and also extended the docs, see https://github.com/home-assistant/developers.home-assistant/pull/2253)